deployment/kubernetes/values.template.yaml

```diff
@@ -1,13 +1,16 @@
-# Change all the below if needed
+# please duplicate template file and rename to "values.yaml" and fill in your value
+
+# change all the below if needed
 PUBLIC_REGISTRATION: false
 INVITE_REGISTRATION: false
 
 BACKEND:
-  # Change all the below if needed
+  # change all the below if needed
   # DOCKER_IMAGE_REPO - change that to your branded docker image
-  #  Label is appended based on .Chart.appVersion
+  #  label is appended based on .Chart.appVersion
```

Comment:
```suggestion
  # label is appended based on .Chart.appVersion
```

Reply:
This is an indent you did, because it belongs to the line above @ulfgebhardt